app/models/user.rb

```diff
@@ -33,6 +33,6 @@ def group_invites(user)
 
   def gravatar_for(user)
     gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
-    "//secure.gravatar.com/avatar/#{gravatar_id}?d=monsterid"
+    "//gravatar.com/avatar/#{gravatar_id}?d=monsterid"
   end
```

Comment:
One little thing: You could remove the argument here and change it to `user.gravatar` from `user.gravatar_for(user)` (see how the user is repeated?).

`user` is accessible inside the method as `self`.